distribution/openrct2.d.ts

```diff
@@ -1246,7 +1245,65 @@ declare global {
         readonly elements: TrackSegmentElement[];
 
         /**
-         * Gets a length of the subpositions list for this track segment.
+         * The curve direction of the suggested following piece, or if the suggested following piece
+         * is a track segment
+         */
+        readonly nextCurve: TrackCurveType;
+
+        /**
+         * The curve direction of the suggested preceding piece, or if the suggested preceding piece
+         * is a track segment
+         */
+        readonly previousCurve: TrackCurveType;
+
+        /**
+         * The track segment for the following piece. Only valid if chainEndCurve is trackSegment.
```

Comment:
I don't think `chainEndCurve` means anything anymore. Can you update this comment?

Reply:
Weird, it's fixed on my end, I thought I pushed